internal/engine/experiment/riseupvpn/riseupvpn_test.go

```diff
@@ -555,6 +568,9 @@ func TestMissingTransport(t *testing.T) {
 	transports = transports[:len(transports)-1]
 	eipService.Gateways[1].Capabilities.Transport = transports
 	eipservicejson, err := json.Marshal(eipservice)
+	if err != nil {
```

Comment:
`staticcheck` warning: `err` is unused (we can otherwise replace `err` with `_`)